### PR TITLE
countdownTimer1.0

### PR DESCRIPTION
**Pull request by JosephRE95**
# Pull Request Checklist

Please make sure you've checked the following items before submitting your pull request:

- [ x] Have you tested your changes locally?
- [ x] Does your app build successfully on your local machine (`npm run build`)?
- [ x] Have you formatted your code according to project standards (`npm run format`)?
- [ x] Have you removed all `console.log` statements?
- [ x] Linked to the GitHub [GitHub issue #6](https://github.com/your-repo/your-project/issues/6)

## Additional Checklist

Please provide the acceptance criteria for this feature and include a link to the corresponding work item or issue.

## Feature Demonstration

Please provide a video demonstrating the functionality of the feature being implemented.

### CODE REVIEW

File: components/countdownTimer.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
```
Comment (Drew1128) [inline components/countdownTimer.tsx:1]: import { useState, useEffect } from 'react';

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Drew1128) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: Great job! Update this file in the GitHub site after committing a PR. Not in this file directly. :)

File: components/countdownTimer.tsx (hunk 1 above)
Comment (Drew1128) [inline components/countdownTimer.tsx:1]:
```suggestion
import { useState, useEffect } from 'react';
```

File: components/countdownTimer.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+// Define the type for the activities array
```
Comment (Drew1128) [inline components/countdownTimer.tsx:3]:
```suggestion
// Define the type for the activities array
```

**Review by Drew1128 — DISMISSED** [review]
Awesome job!

🟡  Remove comments (this is mainly just a preference thing. Every developer you run into have different opinions. In my opinion, I think we should save comments for complex logic that the next developer should know about. This is not a hill. I'm willing to die on. I have really good friends that are great engineers that leave comments everywhere 😄  )

🔴  Don't update the read me template directly. Update that after you submit the PR on github

**Comment by JosephRE95** [comment]
I resolved the issue.

**Review by Drew1128 — APPROVED** [review]

**Pull request merged**